VultisigApp/VultisigAppTests/TestData/ton.json

```diff
@@ -61,6 +63,44 @@
             "vault_public_key_ecdsa": "023e4b76861289ad4528b33c2fd21b3a5160cd37b3294234914e21efb6ed4a452b",
             "lib_type": "DKLS"
         },
-        "expected_image_hash": ["299ecf1da0682c338130b7f3f890484100e41c8f0d2b024b1fd0214df6e310dd"]
+        "expected_image_hash": [
+            "299ecf1da0682c338130b7f3f890484100e41c8f0d2b024b1fd0214df6e310dd"
+        ]
     },
-]
+    {
+        "name": "Send TON USDT Jetton",
+        "keysign_payload": {
+            "coin": {
+                "chain": "Ton",
+                "ticker": "USDT",
+                "address": "UQCc9iCgP_b5RMJcFE5XD8zStfjtNHLhDWfUqC5m1SjSer95",
+                "decimals": 6,
+                "price_provider_id": "tether",
+                "is_native_token": false,
+                "hex_public_key": "6c756400bac0b153b421df6e199302537d12f7d4a53447004485700a958e7571",
+                "contract_address": "EQCxE6mUtQJKFnGfaROTKOt1lZbDiiX1kCixRv7Nw2Id_sDs",
+                "logo": "usdt"
+            },
+            "to_address": "UQDmLe6ticcY_uLZsfurdYONshNuCn8IS81KcJ8p6M6ISMcB",
+            "to_amount": "1000000",
```

Comment:
This test case , you have to keep it exactly the same as android repo , otherwise it defeat the purpose

Please make sure the payload is exactly the same as the one used in android , and make sure test pass